# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.cpp
# Commit: 7b82837e2f56
# Full Hash: 7b82837e2f56582e1e86897285f4109856b72000
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2021-11-11 15:25:44
# Regressor Bug: 1739448
# File Overlap Count: 1
# Description:
#   Bug 1739448 - Implement a prototype WebGL-backed Canvas2D. r=gfx-reviewers,nical,jrmuizel
#   
#   This mainly provides DrawTargetWebgl, which implements the subset of the DrawTarget
#   API necessary for integration with CanvasRenderingContext2D. It translates them to
#   suitable commands for its internal ClientWebGLContext, which then manages remoting
# ==============================================================================

diff -r feac99f1dae3 -r 7b82837e2f56 gfx/thebes/gfxPlatform.cpp
--- a/gfx/thebes/gfxPlatform.cpp	Thu Nov 11 04:33:50 2021 +0000
+++ b/gfx/thebes/gfxPlatform.cpp	Thu Nov 11 04:33:50 2021 +0000
@@ -873,7 +873,7 @@
     // Prefs that don't fit into any of the other sections
     forcedPrefs.AppendPrintf("-T%d%d%d) ",
                              StaticPrefs::gfx_android_rgb16_force_AtStartup(),
-                             0,  // SkiaGL canvas no longer supported
+                             StaticPrefs::gfx_canvas_accelerated(),
                              StaticPrefs::layers_force_shmem_tiles_AtStartup());
     ScopedGfxFeatureReporter::AppNote(forcedPrefs);
   }